lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -754,7 +751,7 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     def valid_candidate?(t), do: ReconstructedAlert.temporarily_override_alert(t)
     def audio_serialize(t), do: ReconstructedAlert.serialize(t)
     def audio_sort_key(t), do: ReconstructedAlert.audio_sort_key(t)
-    def audio_valid_candidate?(_instance), do: true
+    def audio_valid_candidate?(t), do: ReconstructedAlert.temporarily_override_alert(t)
```

Comment:
oh snap! I missed this last time 🤦

Reply:
I almost did this time. Super easy to overlook. Especially because Evergreen allowing audio is fairly new.